libfwupd/fwupd-bios-attr.c

```diff
@@ -22,10 +22,12 @@ fwupd_bios_attr_finalize(GObject *object);
 
 typedef struct {
 	FwupdBiosAttrKind kind;
+	gchar *attr_id;
```

Comment:
hmm, i still think this one wants to be appstream_id.

Reply:
if not, then just `id`

Reply:
It shouldn't be `appstream_id`, I think `id` is fine.